packages/react-core/src/components/Dropdown/DropdownToggle.tsx

```diff
@@ -30,6 +30,8 @@ export interface DropdownToggleProps extends React.HTMLProps<HTMLButtonElement>
   isDisabled?: boolean;
   /** Whether or not the dropdown toggle button should have primary button styling */
   isPrimary?: boolean;
+  /** The image to display in the toggle. */
+  toggleImage?: React.ReactNode | null;
```

Comment:
Why does this need to be null?

Reply:
ReactNode includes null
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/22e6c701e757834e37461dbaad00fb2ae8ea1fc4/types/react/index.d.ts#L241